src/Illuminate/Queue/Events/Sleeping.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Illuminate\Queue\Events;
+
+class Sleeping
+{
+    /**
+     * @var  int
```

Comment:
I almost did and then I removed it. It's a very tiny event class that is called "Sleeping", it has a property that is an integer and is called `seconds`. Isn't it obvious what it does? Description that doesn't provide any new information is not very helpful.